packages/financial-templates-lib/src/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -61,7 +73,7 @@ class DSProxyManager {
     // The user already has a DSProxy deployed. Load it in from the events.
     if (events.length > 0) {
       this.dsProxyAddress = events[events.length - 1].returnValues.proxy;
-      this.dsProxy = new web3.eth.Contract(this.dsProxyAbi, this.dsProxyAddress);
+      this.dsProxy = new this.web3.eth.Contract(this.dsProxyAbi, this.dsProxyAddress);
```

Comment:
this was a bug. worked before as it was in unit tests. this is why we do war games.